photon-api/src/main/scala/com/linkedin/photon/ml/supervised/regression/LinearRegressionModel.scala

```diff
@@ -30,30 +31,69 @@ class LinearRegressionModel(override val coefficients: Coefficients)
   with Regression
   with Serializable {
 
+  /**
+   *
+   * @return The model type
+   */
+  override def modelType: TaskType = LINEAR_REGRESSION
+
+  /**
+   * Compute the mean of the linear regression model.
```

Comment:
"Mean response"? The model gives a response given inputs. This is the mean of the response. The model itself doesn't have a "mean" IMHO, it's just a bunch of coefficients and a link function.